src/plugins/prng/MersenneTwister.cpp

```diff
@@ -0,0 +1,13 @@
+#include "MersenneTwister.h"
+
+MersenneTwister::MersenneTwister(unsigned int seed)
+{
+    std::mt19937 mersenneTwister(seed);
```

Comment:
Since security is not really a concern for us, this is fine.

However, I'd like to add that a Mersenne twister is not cryptographically secure. In fact, if a user can generate a arbitrary amount of passwords and they all use the same seed, he can recover the seed and therefore all passwords that were generated by the twister thus far.

Not saying we need to change that, but this shouldn't be used for a password generator.

Reply:
I understand your concerns. I am however unsure how one would want to recover the seed. Using this implementation the password generator is basically dependent on the (non-)determinism of the [random_device](https://en.cppreference.com/w/cpp/numeric/random/random_device) which is said to produce "non-deterministic random numbers".